.github/workflows/artifacts-ci-dotnet.yml

```diff
@@ -69,6 +74,12 @@ jobs:
       - name: Get repo data
         id: repo_data
         uses: htec-infra/gha-utils/actions/repo-data@v0.3.3
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        if: inputs.platform != 'self-hosted'
```

Comment:
I think you don't need this condition because we have use-cases to assume role for self-hosted runner.